src/audio/AudioManager.cpp

```diff
@@ -78,6 +84,7 @@ AudioManager::AudioManager(){
 *   Description:
 *       Desctructor of AudioManager. Frees memory and quits SLD mixer
 */
+#ifndef SERVER
```

Comment:
Alex why you do this to me?
I wanted you to add these wrappers around the new methods you added since I already did it for the existing ones
This is redundant
Also an fyi, I did it inside the method to prevent any api problems
Better to have an empty call that gets optimized out than a call taht doesn't exist and might break in someone else's code